### ISSUE TITLE
Lacking data is causing doubts

### ISSUE THREAD

**Issue by vtwoptwo** [thehitchhikersguideto/bookworms#2]
Guys .. after several attempts to find a model which would fit our current data set and context... i suggest switching our topic.

1. The frontend will still be used but for a new contextual setting
2. I recommend switching to a review oriented dataset which will help us yield the class contents to its max extend by migrating to the Goodreads api

https://www.goodreads.com/api/terms
https://www.reddit.com/r/webdev/comments/tcbx3m/your_experience_with_goodreads_api/

-> this subreddit mentions other apis but most users seem to be in favour of goodreads (plus i have a membership ;))

I think this would provide us with a  more holistic project trajectory and will allow us to focus on ML rather than data cleaning and generation. 

Let me know what you think?

**Comment by ismadoukkali** [comment]
Found a couple of interesting datasets. The all surround gym/fitness settings but they do not fully serve the purpose of our project. Can we maybe pivot? Still in the same topic/industry (fitness) but with different dataset. What do you think?

https://www.kaggle.com/datasets/open-powerlifting/powerlifting-database
https://www.kaggle.com/competitions/ml-posterior-gym-training-prediction/data
)https://www.kaggle.com/datasets/nithilaa/fitness-analysis

**Comment by vtwoptwo** [comment]
.....I prefer books 0.0

**Comment by ismadoukkali** [comment]
Let's definitely talk about it more... I started some scraping to generate a larger dataset of exercises.

These are individual exercises mapped to existing workout programs. 
https://docs.google.com/spreadsheets/d/1LO6Xi9EY2ppDsbSajbN7_Dne_KUML4tlUoZc6k41AI8/edit?usp=sharing 

I extracted all the exercises from these programs.
https://drive.google.com/drive/folders/1WMMjCEwCDIGQcrZkv9qgcBZJmRkICbWZ?usp=share_link
